twake/frontend/src/app/scenes/Apps/Messages/Input/Input.tsx

```diff
@@ -139,6 +151,7 @@ export default (props: Props) => {
       {!hasEphemeralMessage && !props.messageId && (
         <InputOptions
           inputValue={content}
+          isEmpty={!!content || !hasFilesAttached}
```

Comment:
This is wrong to me, the correct line should be
-> isEmpty={!content && !hasFilesAttached}
or
-> isEmpty={!(content || hasFilesAttached)}

Reply:
Fixed